ios/RNCPagerScrollView.m

```diff
@@ -1,26 +0,0 @@
-
-#import "RNCPagerScrollView.h"
-
-@implementation RNCPagerScrollView
-- (instancetype)initWithFrame:(CGRect)frame {
-    if (self = [super initWithFrame:frame]) {
-        self.panGestureRecognizer.delegate = self;
-    }
-    return self;
-}
-
-- (BOOL)gestureRecognizer:(UIGestureRecognizer *)gestureRecognizer shouldRecognizeSimultaneouslyWithGestureRecognizer:(UIGestureRecognizer *)otherGestureRecognizer {
```

Comment:
Why this was deleted?

Reply:
Could you comment on that code and post the above comment in the code? 🙏

Reply:
Sounds good!

Reply:
@troZee done 👍